templates/primary-secondary/primary-secondary.js

```diff
@@ -52,9 +58,21 @@ class TemplatePrimarySecondary extends LitElement {
 	}
 
 	render() {
+		const containerStyles = {};
+		const navigationStyles = {};
+		if (this.width) {
+			containerStyles.maxWidth = `${this.width}px`;
+			navigationStyles['--d2l-navigation-immersive-content-max-width'] = `${this.width}px`;
+		}
+
 		return html`
-			<div class="container">
-				<header><slot name="header"></slot></header>
+			<div class="container" style="${styleMap(containerStyles)}">
```

Comment:
It looks like this would constrain everything, including the header and its contents.  In the above code, I _think_ it would have to be `containerStyles['max-width'] = ...`, unless the styleMap directive does some automagical conversion.

More importantly, I don't think the page template should have nav baked into it.  Rather, I think the max-width should just be applied to the non-header elements, but the consumer would have to apply similar outside of this component for `--d2l-navigation-immersive-content-max-width`. Maybe other solutions, but I don't think core should be sucking in nav.

Reply:
The `lit-html` docs show it to be camel-case.

Yep, I've removed the navigation and will continue to pass it in the `header` `slot`.